docs/notebooks/ordinal_regression.ipynb

```diff
@@ -0,0 +1,1798 @@
+{
```

Comment:
<p>I know I'm being very nitpicky, but it can help users when they're reading this. There is no "ordered distribution", so I would say "recover the parameters of an ordinal model". You don't need to add the following, but I think the explanation is that the response distribution is not ordered. It's just a categorical distribution with its parameter vector built in some special way. This special way is that there is a parameter vector that goes through some transformation to obtain an ordered parameter vector.</p><p> </p><p>Again, not cumulative link function, but cumulative family. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/719/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Yeah, that could be interpreted wrong. Thanks!</p>